File: wts0022_lab01/wts0022_lab01.java
import java.util.Scanner;
import java.io.File;

public class wts0022_lab01{
    public static int getSize(String path){
        File dir = new File(path);
        int total = 0;

        for (String elem : dir.list()){
            File f = new File(path + "/" + elem);
            if(f.isFile())
                total += f.length();
            else
                total += getSize(path + "/" + elem);
        }
        return total;
    }
    public static void main(String[] args){
        String currentDir = System.getProperty("user.dir");
        System.out.println(getSize(currentDir) + " bytes");
    }
}

File: wts0022_lab01/wts0022_lab01.py
#  William Sigala
#  CSE 3302
#  Lab 1
#  ID# 1001730022

#  compile: python3 wts0022_lab01.py

import os

def get_size(path):
    total = 0
    
    for elem in os.listdir(path):
        if os.path.isfile(f"{path}/{elem}"):
            total += os.path.getsize(f"{path}/{elem}")
        else:
            total += get_size(f"{path}/{elem}")
    return total

total_size = get_size(os.getcwd())
print(f"{total_size} bytes")